projects/mac_ai/testing/scripts/podman_start_machine.api_remoting.sh

```
@@ -13,8 +13,14 @@ echo "INFO: # for podman to load krunkit/libkrun and not vfkit"
 echo "INFO: Setting CONTAINERS_MACHINE_PROVIDER=$CONTAINERS_MACHINE_PROVIDER"
 echo
 
+if [[ ! podman machine info >/dev/null ]]; then
```

Shellcheck (0.10.0)

[error] Couldn't parse this test expression. Fix to allow more checks.

(SC1073)